server/src/test/java/org/opensearch/indices/replication/SegmentReplicationTargetServiceTests.java

```diff
@@ -48,12 +49,6 @@ public void setUp() throws Exception {
         sut = new SegmentReplicationTargetService(threadPool, recoverySettings, transportService, replicationSourceFactory);
     }
 
-    @Override
-    public void tearDown() throws Exception {
-        closeShards(indexShard);
```

Comment:
Nitpick - rather than remembering to call `closeShards` on all tests, I think it would be more reliable to change the check here:

https://github.com/opensearch-project/OpenSearch/blob/fb1375987643f7c99ae77442401caffca5dddc25/test/framework/src/main/java/org/opensearch/index/shard/IndexShardTestCase.java#L691

to 

```
if (shard != null && && shard.state != IndexShardState.CLOSED)
```

Reply:
Got it, actually this condition `if (shard != null && && shard.state != IndexShardState.CLOSED)` is not necessary just putting back teardown() works fine. Thanks for catching this @kartg